### ISSUE TITLE
Option to not group windows

### ISSUE THREAD

**Issue by Majki** [micheleg/dash-to-dock#108]
I would love to see option for not grouping windows. Unity for Ubuntu groups them for example. And extra one small dot is not always most readable indicator of having 2 windows opened at the same time.

**Comment by ticpu** [comment]
This demand really sounds Frippery Bottom Panel / like https://extensions.gnome.org/extension/3/bottom-panel/

**Comment by gtirloni** [comment]
This would be awesome.

**Comment by dinosaurfiles** [comment]
Yes, I would also like to have this feature

**Comment by philroche** [comment]
+1 for this.

**Comment by hitok** [comment]
+1

**Comment by nediml** [comment]
+1

**Comment by Hronom** [comment]
+1

**Comment by chrisjbillington** [comment]
Frippery bottom panel is close to this, but does not support multiple monitors.

**Comment by localdevjs** [comment]
Just adding my voice to this feature request, it would be an awesome enhancement!

**Comment by sevenjay** [comment]
the great one could be

**Comment by peb-adr** [comment]
+1

**Comment by chrisjbillington** [comment]
Lol, dash to panel has this feature, but also merges with the top panel (optionally moving it to the bottom).

There is a feature request on dash to panel to make it use its own panel instead of merging with the existing one.

And there's this feature request to make dash to dock be more like a traditional panel.

If both are implemented, the projects will be getting pretty close to having the same feature set which would be pretty funny. Maybe in the long run they should be the same project.

**Comment by lwbaqueros** [comment]
I agree, hopefully both come together, and users can configure both a dock and a top bar as part of just one gnome-extention. 
By the way, having `ungroup apps` option in Dash to Dock would be awesome to speed up productivity.

**Comment by djboni** [comment]
+1
It would be nice to have the option to not group the windows, so we can click directly on what we need.
The possibility to reorder the group, or reorder all applications in the taskbar, would also be very welcome.

**Comment by aindriu80** [comment]
Ungrouping applications would be a killer feature, right-clicking on a program to select the second instance is very uncomfortable. I've a dual monitor display and having one icon for several instances can be very confusing.

**Comment by sumancbd** [comment]
+1
Please can we have this feature? Or is there any config I can change using dconf?
Thankyou

**Comment by ant0nwax** [comment]
+1
Did microsoft actually add a feature like this to their current operating sXstem? Or did microsoft still not add a feature like this to windows 11?

**Comment by surajhpatil** [comment]
+1 as well!

**Comment by igiona** [comment]
+1, having the windows grouped it's a pain when working with multiple monitors

**Comment by hsiktas** [comment]
Just to add to this, ungrouping should not mean that the title bar text has to be shown as well.
I use the program "7+ Taskbar Tweaker" on Windows 10 to achieve something like this:

![Image](https://github.com/user-attachments/assets/abca49c4-cc29-41eb-a5f5-c54083467bdb)

Each Firefox window or PDF document in Evince could show its own icon, and on hover, it would pop in the window title and thumbnail preview.
I was trying to work with a stock Ubuntu experience for a few days, but the current implementation is so limited that I had to switch to Dash to Panel for now.

**Comment by xmha97** [comment]
+1 for this.

**Comment by kenoiobi** [comment]
+1 for this

**Comment by ayell** [comment]
+1 plz

**Comment by vidneu** [comment]
+1

**Comment by jakubthedeveloper** [comment]
+7

**Comment by bernardosts** [comment]
Please implement this guys.
while true:
  👍 +1

**Comment by ant0nwax** [comment]
> +1 Did microsoft actually add a feature like this to their current operating sXstem? Or did microsoft still not add a feature ass this to windows 11?

even smallcutie (bill gates and charles simonyi's business) had been able to reinvent the ungroup feature after windows 10 again in the windows 11 2nd ed.

For those who suffer on wayland or x11,
i found LXQT for ubuntu as window manager
and there ungroup exisits

it is also more lightweight.
please do not reply, install lxqt and close this topic. noone has the means to invent ungroup for ubuntu desktop.

**Comment by rasyidrafi** [comment]
this is a simple feature, and its been damn many years hasnt been implemented,
im using dash to dock and frustated having this issues, the mantainer didnt response, so i coded it myself, since this just a gnome extension the code isnt that complicated isnt.

ive working implementation here https://github.com/micheleg/dash-to-dock/pull/2526
i hope the mantainer will to merge my pull request, or provide feedback.

this works well with basic ungrouping

## Example
<img width="715" height="137" alt="Image" src="https://github.com/user-attachments/assets/aeb080a9-b882-4f76-90eb-bdb09adefc97" />
  
## Settings
<img width="1006" height="601" alt="Image" src="https://github.com/user-attachments/assets/847aa931-304d-40dd-8154-c9626aebcec8" />
